### DISCUSSION TITLE
ADX getting NaN values

### DISCUSSION THREAD

**Discussion by myalgomate** [DaveSkender/Stock.Indicators#857] [Help and support]
Hi,
I hope you are doing well here i am attaching the OHLC data and ADX data for 3 min time frame.
I am using Heiken Ashi data to calculate ADX with 14 periods.

I am getting Nan value.

[HeikinAshiData.csv](https://github.com/DaveSkender/Stock.Indicators/files/9248312/HeikinAshiData.csv)

[ADX.csv](https://github.com/DaveSkender/Stock.Indicators/files/9248314/ADX.csv)

**Comment by DaveSkender** [comment]
See #859 to track issue

**Reply by DaveSkender** [reply]
Fixed in v2.1.2
